src/engine/renderer/tr_backend.cpp

```diff
@@ -2769,7 +2769,20 @@ void RB_RenderPostDepth()
 	vec3_t zParams;
 	int w, h;
 
-	GLimp_LogComment( "--- RB_RenderPostDepth ---\n" );
+	GLimp_LogComment( "--- RB_RenderPostDepthLightTile ---\n" );
+
+	/* Do not run lightTile code when the tiled renderer is not used.
+
+	Especially since the related GLSL code cannot run on the small
```

Comment:
It would be more relevant to say why the code accomplishes nothing/is not needed when the tiled renderer is not used, rather than some information about old hardware. It goes without saying that it is good to skip the code if it doesn't do anything.